legal-api/report-templates/amalgamationApplication.html

```diff
@@ -1,13 +1,241 @@
 [[macros.html]]
-
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-      <title>Amalgamation Application</title>
-      <meta charset="UTF-8">
-      <meta name="author" content="BC Registries and Online Services">
-      [[common/style.html]]
-    </head>
-    <body>
-    </body>
+  <head>
+    <title>Amalgamation Application</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="BC Registries and Online Services">
+    [[common/style.html]]
+  </head>
+  <body>
+    <div class="header">
+      <table class="header-table" role="presentation">
+        <tr class="no-page-break">
+          <td>
+            [[logo.html]]
+          </td>
+          <td>
+            <div class="report-type">AMALGAMATION APPLICATION</div>
+            <div class="report-type-desc">BC Limited Company - Business Corporations Act</div>
```

Comment:
Do this https://github.com/bcgov/lear/blob/main/legal-api/report-templates/incorporationApplication.html#L21

Reply:
Looks a lot better, thanks!